pl-PL.json

```diff
@@ -1,6 +1,15 @@
 {
-  "0.3% LP Fee": "0.3% LP Opłaty",
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
+  "A collateral is required in order to interact with smart contracts on Cardano. The collateral amount should be greater than": "A collateral is required in order to interact with smart contracts on Cardano. The collateral amount should be greater than",
+  "Collateral not found!": "Collateral not found!",
```

Comment:
@baymac  Are those already working (are respected by translation engine/website)?

Reply:
yes, but the upstream PR isn't merged yet so this PR hasn't also been merged